docs/en_US/Compressor/CompressionUtils.md

```diff
@@ -0,0 +1,109 @@
+
+### Usage
+
+Following codes show the basic usage of the SensitivityAnalysis.
+```python
+from nni.compression.torch.utils.sensitivity_analysis import SensitivityAnalysis
+
+def val(model):
+    model.eval()
+    total = 0
+    correct = 0
+    with torch.no_grad():
+        for batchid, (data, label) in enumerate(val_loader):
+            data, label = data.cuda(), label.cuda()
+            out = model(data)
+            _, predicted = out.max(1)
+            total += data.size(0)
+            correct += predicted.eq(label).sum().item()
+    return correct / total
+
+s_analyzer = SensitivityAnalysis(model=net, val_func=val)
+sensitivity = s_analyzer.analysis(val_args=[net])
+os.makedir(outdir)
+s_analyzer.export(os.path.join(outdir, filename))
+```
+
+Two key parameters of SensitivityAnalysis are model, and val_func. model is the neural network that to be analyzed and the val_func is the validation function that returns the model accuracy/loss/ or other metrics on the validation dataset. Due to different scenarios may have different ways to calculate the loss/accuracy, so users should prepare a function that returns the model accuracy/loss on the dataset and pass it to SensitivityAnalysis.
+SensitivityAnalysis can export the sensitivity results as a csv file usage is shown in the example above.
+
+Futhermore, users can specify the sparsities values used to prune for each layer by optinal parameter 'sparsities'.
+```python
+s_analyzer = SensitivityAnalysis(model=net, val_func=val, sparsities=[0.25, 0.5, 0.75])
+``` 
+the SensitivityAnalysis will prune 25% 50% 75% weights gradually for each layer, and record the model's accuracy at the same time (SensitivityAnalysis only prune a layer once a time, the other layers are set to their original weights). If the sparsities is not set, SensitivityAnalysis will use the numpy.arange(0.1, 1.0, 0.1) as the default sparsity values.
+
+Users can also speed up the progress of sensitivity analysis by the early_stop_mode and early_stop_value option. By default, the SensitivityAnalysis will test the accuracy under all sparsities for each layer. In contrast, when the early_stop_mode and early_stop_value are set, the sensitivity analysis for a layer will stop, when the accuracy/loss has already meet the threshold set by early_stop_value. We support four early stop modes:  minimize, maximize, dropped, raised.
+minimize: The analysis stops when the validation metric return by the val_func lower than early_stop_value.
+maximize: The analysis stops when the validation metric return by the val_func larger than early_stop_value.
+dropped: The analysis stops when the validation metric has dropped by early_stop_value.
+raised: The analysis stops when the validation metric has raised by early_stop_value.
```

Comment:
better to itemize these four values. you can check the rendered version.

Reply:
get confused about `dropped` and `raised`. what is the difference between `dropped`/`raised` and `minimize`/`maximize`?

Reply:
dropped is used in the cases that you want to stop when the accuracy has dropped by 5%, for example.
minimize is used in the cases that you want to keep the accuracy larger than a specified threshold during the analysis.